content/en/docs/setup/release/version-skew-policy.md

```diff
@@ -28,8 +28,8 @@ The Kubernetes project maintains release branches for the most recent three mino
 
 Applicable fixes, including security fixes, may be backported to those three release branches, depending on severity and feasibility.
 Patch releases are cut from those branches at a regular cadence, or as needed.
-This decision is owned by the [patch release manager](https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/patch-release-team.md#release-timing).
-The patch release manager is a member of the [release team for each release](https://github.com/kubernetes/sig-release/tree/master/release-team).
+This decision is owned by the [patch release team](https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/patch-release-team.md#release-timing).
```

Comment:
It's perhaps worth noting here these two additional links:

 * Patch release schedule: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md
 * Cherry pick process (including patch review criteria): https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md

Reply:
Is it fine if we just add it as "For more information, see Kubernetes Patch releases" and link it to https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md
as it already has schedule, cherry pick process and other information related to patch release.